### PR TITLE
1.1.0 Changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Added full screen images
- Rewrote setup, readme, and Usage
- Annotated screenshots

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Release Information
+
+- **Version**: 1.0.0
+
+- **Certified**: Yes
+
+- **Publisher**: Fortinet  
+
+- **Compatibility**: 7.3.1 and later
```
Comment (cs-abhinav-goel) [inline README.md]: @cs-abhishek-shukla : Please change the FSR compatibility as 7.4.3 & later

File: docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Usage
+
+The **Playbook Execution Wizard** widget renders playbook execution logs and related comments on the wizard for a more fine-grained control on your processes.
+
+This wizard can currently with used in tandem with following widgets:
```
Comment (cs-abhinav-goel) [inline docs/usage.md]: @cs-abhishek-shukla : Please rephrase the sentence. Grammatically looks incorrect

**Review by cs-abhinav-goel — CHANGES_REQUESTED** [review]
@cs-abhishek-shukla : Please do the changes

File: docs/usage.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+1. JSON to Grid Widget
```
Comment (cs-abhinav-goel) [inline docs/usage.md:10]: Can we give doc hyperlink of these widget also.
@cs-abhishek-shukla

File: README.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply README.md]: Done

File: docs/usage.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Fixed

File: docs/usage.md (hunk 3 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md:10]: Docs are in separate repositories and hence linking may break the doc flow

**Review by cs-sujeet-ahirrao — APPROVED** [review]

**Pull request merged**